### PR TITLE
Added web kubectl in cluster list page and removed type agent columns

### PR DESCRIPTION
**Pull request by niravparikh05**
### What does this PR change?

- Added web kubectl in cluster list page and removed type agent columns from audit logs

### Does the PR depend on any other PRs or Issues? If yes, please list them.

- No

### Checklist

I confirm, that I have...

- [x] Read and followed the contributing guide in `CONTRIBUTING.md`
- [ ] Added tests for this PR
- [x] Formatted the code using `npm run format` (if applicable)
- [ ] Updated [documentation on the Paralus docs site](https://github.com/paralus/website/blob/main/docs) (if applicable)
- [x] Updated `CHANGELOG.md`

### CODE REVIEW

File: src/app/routes/edges/routes/PrivateEdgeList/index.js [hunk 1]
```diff
@@ -1314,37 +1341,45 @@ class PrivateEdgeList extends React.Component {
...
                           <TableCell data={n}>
-                            {this.state.userRole !== "READ_ONLY_OPS" && (
-                              <Tooltip title="Kubectl Settings">
-                                <IconButton
-                                  aria-label="edit"
-                                  className="m-0"
-                                  onClick={(event) => {
-                                    this.handleKubectlSettings(null, n);
+                            <div className="row">
+                            { isClusterReady(n) && (
```
Comment (akshay196) [inline src/app/routes/edges/routes/PrivateEdgeList/index.js:1345]: Should we add `this.state.userRole !== "READ_ONLY_OPS"` check similar to Setting and Delete action?

File: src/app/routes/edges/routes/PrivateEdgeList/index.js [hunk 2]
```diff
@@ -1314,37 +1341,45 @@ class PrivateEdgeList extends React.Component {
...
-                            {this.state.userRole !== "READ_ONLY_OPS" && (
-                              <Tooltip title="Kubectl Settings">
-                                <IconButton
-                                  aria-label="edit"
-                                  className="m-0"
-                                  onClick={(event) => {
-                                    this.handleKubectlSettings(null, n);
+                            <div className="row">
+                            { isClusterReady(n) && ( 
+                                <KubeCtlShellAccess
```
Comment (akshay196) [inline src/app/routes/edges/routes/PrivateEdgeList/index.js:1346]: Seem like `KubeCtlShellAccess` function render `>_ KUBECTL` to the UI. If yes, should we just show icon so it is consistent with other actions like "Delete" and "Setting"? Add "Web kubectl" as tooltip to this icon?

File: src/components/KubeCtlShellAccess.js [hunk 3]
```diff
@@ -16,6 +16,7 @@ const useStyles = makeStyles((theme) => ({
     marginRight: "5px",
     marginTop: "4px",
     lineHeight: "10px",
+    height: "25px"
```
Comment (akshay196) [inline src/components/KubeCtlShellAccess.js:19]: This is looking weird on cluster view page:
Screenshot from 2022-10-07 12-03-48

**Review by akshay196 — CHANGES_REQUESTED** [review]
I didn't tested this PR myself, but it seems like it will work. Just few minor UI changes:

File: src/app/routes/edges/routes/PrivateEdgeList/index.js (hunk 1 above)
Comment (niravparikh05) [reply src/app/routes/edges/routes/PrivateEdgeList/index.js:1345]: not required, sentry jit will take care of translating paralus read roles to corresponding role or clusterrole rbac and user will be able to perform view only kubectl commands.

File: src/app/routes/edges/routes/PrivateEdgeList/index.js (hunk 2 above)
Comment (niravparikh05) [reply src/app/routes/edges/routes/PrivateEdgeList/index.js:1346]: yeah makes sense, modified accordingly.

File: src/components/KubeCtlShellAccess.js (hunk 3 above)
Comment (niravparikh05) [reply src/components/KubeCtlShellAccess.js:19]: reduced the height to 20 so that it looks consistent.

**Review by akshay196-rafay — APPROVED** [review]

**Review by akshay196 — APPROVED** [review]
@nirav-rafay Please add an entry for relogin issue fix in CHANGELOG.md

**Comment by niravparikh05** [comment]
relogin issue fix has been removed from this pull request, needs to be further analyzed and come up with an appropriate fix.

**Review by akshay196 — APPROVED** [review]

**Pull request merged**